packages/react-native-tab-view/src/PagerViewAdapter.tsx

```diff
@@ -35,8 +35,6 @@ type Props<T extends Route> = PagerProps & {
   ) => React.ReactElement;
 };
 
-const useNativeDriver = Platform.OS !== 'web';
```

Comment:
Also tested on web, it is using `PanResponderAdapter` there so i guess this change should be fine. but let me know!